WebKitBrowser/Extension/main.cpp

```diff
@@ -201,13 +201,15 @@ static class PluginHost {
                 G_CALLBACK(didStartProvisionalLoadForFrame), nullptr);
 #endif
     }
+PUSH_WARNING(DISABLE_WARNING_DEPRECATED_USE)
```

Comment:
PUSH_WARNING / POP_WARNING  doesn't seem to be available in Thunder R2. We still have to build with 3 versions of Thunder: R2, R4.2 and R4.4. Could you please check if this will work with all 3 versions of Thunder?

Reply:
Sorry, I was not aware we need this to work with Thunder R2, and you are correct that `PUSH_WARNING / POP_WARNING` are no available on R2.

In that case, I think we have two options here, either implement the `PUSH_WARNING / POP_WARNING` like it is in Portability here: https://github.com/rdkcentral/Thunder/blob/283b3d54334010403d85a4e69b3835de23e42331/Source/core/Portability.h#L100, or if the build does not need to be warning free, then we can just remove it. Thoughts @modeveci?

Reply:
@VeithMetro, for compatibility, it is better to get @MFransen69 feedback.

Reply:
Push/Pop warning checking may be removed from this PR, because the deprecated APIs (from wpe 2.40 until 2.45) brought back. As we do not have direct usage of any versions between 2.40-2.45. it may be good to not put any checks. Let me first confirm and then we can move on.

Reply:
@VeithMetro , we can remove POP/PUSH macros, I got confirmation! 2.38 and 2.46 will not give any warnings. 
Thank you very much! ;)